packages/sources/galaxy/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@chainlink/galaxy-adapter",
+  "version": "0.2.10",
```

```suggestion
  "version": "0.0.0",
```
Automation on this coming this week.